README.md

```diff
@@ -1,39 +1,277 @@
+```
+
+3. Build project
+
+```bash
+make build
+```
+
+4. Build image
+
+```bash
+make docker-build
+```
+
+5. Push image to registry
+
+<div tabs name="Push image" group="application-connector-installation">
+  <details>
+  <summary label="k3d">
+  k3d
+  </summary>
+
+   ```bash
+   k3d image import $IMG -c $K3D_CLUSTER_NAME
+   ```
+  </details>
+  <details>
+  <summary label="Docker registry">
+  Globally available Docker registry
+  </summary>
+
+   ```bash
+   make docker-push
+   ```
+
+  </details>
+</div>
+
+6. Deploy
+
+```bash
+make deploy
+```
+
+## Using `application-connector-manager`
+
+- Create ApplicationConnector instance
+
+```bash
+kubectl apply -f config/samples/operator_v1alpha1_applicationconnector.yaml
+```
+
+- Delete ApplicationConnector instance
+
+```bash
+kubectl delete -f config/samples/operator_v1alpha1_applicationconnector.yaml
+```
+
+- Update ApplicationConnector properties
+
+TODO: Provide example of CR update
+
+## Build and install of Application Connector module in modular Kyma on the local k3d cluster
+
+1. Setup local k3d cluster and local Docker registry
+
+```bash
+k3d cluster create kyma --registry-create registry.localhost:0.0.0.0:5001
+```
+2. Add the `etc/hosts` entry to register the local Docker registry under the `registry.localhost` name
+
+```
+127.0.0.1 registry.localhost
+```
+
+3. Export environment variables (ENVs) pointing to module and the module image registries
+
+```bash
+export IMG_REGISTRY=registry.localhost:8888/unsigned/operator-images
+export MODULE_REGISTRY=registry.localhost:8888/unsigned
+```
+
+4. Build Application Connector module 
+```bash
+make module-build
+```
+
+This command builds an OCI image for Application Connector module and pushes it to the registry and path, as defined in `MODULE_REGISTRY`.
+
+5. Build Application Connector manager image
+```bash
+make module-image
+```
+
+This command builds a Docker image for Application Connector Manager and pushes it to the registry and path, as defined in `MODULE_REGISTRY`.
+
+6. Verify if the module and the manager's image are pushed to the local registry
+
+```bash
+curl registry.localhost:8888/v2/_catalog
+{"repositories":["unsigned/component-descriptors/kyma.project.io/module/application-connector","unsigned/operator-images/application-connector-operator"]}
+```
+
+7. Inspect the generated module template
+
+The following are temporary workarounds.
+
+Edit the `template.yaml` file and:
+
+- change `target` to `control-plane`
+>**NOTE:** This is only required in the single cluster mode
+
+```yaml
+spec:
+  target: control-plane
+```
+
+- change the existing repository context in `spec.descriptor.component`:
+>**NOTE:** Because Pods inside the k3d cluster use the docker-internal port of the registry, it tries to resolve the registry against port 5000 instead of 8888. K3d has registry aliases but module-manager is not part of k3d and thus does not know how to properly alias `registry.localhost:8888`
```

Comment:
```suggestion
>**NOTE:** Because Pods inside the k3d cluster use the docker-internal port of the registry, it tries to resolve the registry against port 5000 instead of 8888. K3d has registry aliases, but module-manager is not part of k3d and thus does not know how to properly alias `registry.localhost:8888`.
```